src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -90,15 +92,16 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
     response.setContentType("application/json");
 
-    String user = request.getParameter("user");
+    //String user = request.getParameter("user");
+    String postID = request.getParameter("postID");
```

Comment:
There are quite a few lines that the user parameter. 
https://github.com/ellikap6/CodeU-team11/blob/master/src/main/webapp/js/user-page-loader.js#L125
https://github.com/ellikap6/CodeU-team11/blob/master/src/main/webapp/js/user-page-loader.js#L150
https://github.com/ellikap6/CodeU-team11/blob/master/src/main/webapp/js/user-page-loader.js#L53

Instead of changing all these lines now (I mean you can if you want, but I think the AboutMe change will be a little confusing.) you can just do
`String postID = request.getParameter("user");`
here for now and then leave the changing of the routing of that parameter to this file for later.

Reply:
IF you use user here, our app should still work.